code/sound/sound.cpp

```diff
@@ -737,6 +743,10 @@ sound_handle snd_play_3d(game_snd* gs, vec3d* source_pos, vec3d* listen_pos, flo
 	}
 
 	if (handle.isValid()) {
+		if (looping) {
+			currentlyLooping3dSoundInfos.emplace_back(handle, default_volume, vol_scale);
```

Comment:
These parameters were copied from the existing call in `snd_play()` for `currentlyLoopingSoundInfos`. I'm not sure if these are the correct parameters to use here.